crates/sqllib/src/timestamp.rs

```diff
@@ -1244,14 +1244,14 @@ some_polymorphic_function2!(datediff_day, Date, Date, Date, Date, i32);
 
 #[doc(hidden)]
 pub fn format_date__(format: SqlString, date: Date) -> SqlString {
-    return SqlString::from(date.to_dateTime().format(&format.str()).to_string());
+    return SqlString::from(date.to_dateTime().format(format.str()).to_string());
```

Comment:
Is this `to_string()` needed?

Reply:
format returns some strange data type called `DelayedFormat`.